pallets/balances/src/lib.rs

```diff
@@ -1329,6 +1319,85 @@ where
     }
 }
 
+/// Additional functionality atop `LockableCurrency` allowing a local,
+/// per-id, stacking layer atop the overlay.
+pub trait LockableCurrencyExt<AccountId>: LockableCurrency<AccountId> {
```

Comment:
It is better to keep the trait declaration under `common/traits/balances` for code cleanliness purposes.